lib/ansible/modules/cloud/docker/docker_node_facts.py

```diff
@@ -29,8 +29,19 @@
   name:
     description:
       - The name of the node to inspect.
+      - The list of nodes names to inspect.
+      - If empty then return information of all nodes in Swarm cluster.
       - When identifying an existing node name may either the hostname of the node (as registered in Swarm) or node ID.
-    required: true
+      - If I(self) is C(True) then this parameter is ignored.
+    required: false
+  self:
+    description:
+      - If C(True) then query the host module run on or one specified as I(docker_host).
+      - If C(True) then I(name) is ignored.
```

Comment:
In Ansible, `true` or `yes` is preferred over other boolean forms. (Same for `false`/`no`.)

([Reference](https://github.com/ansible/community/issues/389#issuecomment-452327393))

Reply:
It says both yes/no and true/false are accepted with no preference to any in documentation and examples. I don't mind changing it of course to yes/no here.

I also noticed in `docker_swarm` module there is inconsistency in using boolean values in documentation vs. examples

Reply:
You're currently using `True`, which is neither `true` nor `yes`. You can use either `true` or `yes`, but not `True`.

Reply:
The capital letters in values as per your other comments were already amended, I got your comment wrong

Reply:
Ok, no problem :)